lib/reporter/results.js

```diff
@@ -212,7 +213,8 @@ module.exports = class Results {
     }
 
     suiteResults.results.lastError = lastError;
-    suiteResults.results.skipped = this.skipped;
+    suiteResults.results.skippedAtRuntime = this.skipped;
```

Comment:
Should we rename this.skipped to `skippedAtRuntime` to avoid any future confusion?

Reply:
yes I think we should, but we must keep `results.skipped` otherwise it will break existing reporters